test/e2e/next-image/app/pages/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import Image from 'next/image'
+import { motion } from 'framer-motion'
+
+const CustomImage = React.forwardRef((props, ref) => (
+  <Image
+    ref={ref}
+    id="img"
+    src="www.fillmurray.com/300/300"
```

Comment:
```suggestion
    src="https://www.fillmurray.com/300/300"
```

Reply:
Sorry I need to fix my GitHub notifications.
FYI links to 3rd party domains are not working in tests (which is reasonable), I forgot to use a committed image here.